hvpy/utils.py

```diff
@@ -15,11 +15,13 @@ def decorator(func):
     return decorator
 
 
-def convert_date_to_isoformat(v: datetime) -> str:
+def convert_date_to_isoformat(v: datetime) -> Union[str, None]:
     """
     Converts the date from a datetime object to a string in the ISO format.
     """
-    return v.isoformat() + "Z"
+    if v:
+        return v.isoformat() + "Z"
+    return None
```

Comment:
```suggestion
    if v is not None:
        return v.isoformat() + "Z"
```
What happens if this is the code?

Reply:
Yes, same output but better code.